src/openni2/DeviceDriver.cpp

```diff
@@ -500,7 +500,43 @@ namespace Freenect2Driver
             WriteMessage("Opening device " + std::string(uri));
```

Comment:
Not necessary, it is already included via `#include <libfreenect2/libfreenect2.hpp>`, so it compiled as is.